### ISSUE TITLE
[RFE] FindMissing and show a sample output

### ISSUE THREAD

**Issue by ukdtom** [WebTools-NG/WebTools-NG#624]
Could this be included in the Wiki?

https://github.com/WebTools-NG/WebTools-NG/blob/master/wiki/modules/exporttools/FindMissing-Output.png

**Comment by trumpy81** [comment]
The image is too wide for the Wiki, which would mean it would display very small and unreadable.

What specifically do you want to show?

**Comment by ukdtom** [comment]
Just what the output looked like, so people could better understand what it did

**Comment by ukdtom** [comment]
Maybe like this?

https://github.com/ukdtom/testvid/wiki

**Comment by trumpy81** [comment]
Done, see the bottom of [Find Missing Episodes](https://github.com/WebTools-NG/WebTools-NG/wiki/FindMissingEpisodes)

**Comment by trumpy81** [comment]
I tried to `Find Missing Episodes` from your QNAP7 server, but the .csv is all but empty. It only lists the Key and the Title.

[QNAP7_TV Shows_TV_TV Shows_Find Missing Episodes_Item 0-269_2022.11.05_4.56.26.csv](https://github.com/WebTools-NG/WebTools-NG/files/9941319/QNAP7_TV.Shows_TV_TV.Shows_Find.Missing.Episodes_Item.0-269_2022.11.05_4.56.26.csv)

I wanted to recapture the image showing the Guid problem and possibly an episode mismatch, so I could comment on them.

I tried using my own library, but as luck would have it, I don't have any Guid or episode mismatches ... lol
